lib/examples/reboot.js

```diff
@@ -0,0 +1,17 @@
+// transpile:main
+import { connectToDevice } from '../../index.js';
+import { CONNECTION_TYPES } from '../constants';
+
+async function start () {
+  let adb = connectToDevice(CONNECTION_TYPES.USB);
+  await adb.start();
+  console.log("device started");
+  let command = {
+    type: "reboot"
+  };
+  await adb.open(command);
+  console.log("device should reboot now");
+}
+
+console.log("starting");
+start();
```

Comment:
this will swallow any errors, not sure if you care about that. i wrote a little util called `asyncify` in asyncbox that prints out errors.

```js
import { asyncify } from 'asyncbox';
...
asyncify(start);
```